client/components/Donate/DonateSelectPaymentType.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import ValidatedCurrencySelect from '../ValidatedCurrencySelect'
+
+
+const INVALID_CURRENCY_MESSAGE = 'Currency is Required'
+const initialState = {
```

Comment:
unless you need to reinitialize component state, just define state members within the class property.

Reply:
I have fixed this. :D

Reply:
I like trains